src/components/Dashboard/TodoList.vue

```diff
@@ -1216,16 +1209,16 @@ export default class TodoList extends Vue {
         paymentToken: header.paymentToken || null,
         payErrorObj,
         // FUTURE: ideally, this would come from the filing:
-        warnings: this.getBusinessWarnings.map(warning => warning.message)
-      }
+        warnings: (this.getBusinessWarnings as any).map(warning => warning.message)
```

Comment:
This is done to fix the `Property 'map' does not exist on type 'BusinessWarningIF'` eslint error.

Reply:
The root problem here is that `getBusinessWarnings` is incorrectly typed in the `@Getter`... it should be an array, and then the previous code here should be fine.

Please fix and revert this line and line 1242 below.

Reply:
Fixed, thanks.